usb/backend/libusb0.py

```diff
@@ -510,6 +512,9 @@ def get_configuration(self, dev_handle):
         assert ret == 1
         return buff[0]
 
+    @methodtrace(_logger)
+    def set_interface_altsetting(self, dev_handle, intf, altsetting):
+        _check(_lib.usb_set_altinterface(dev_handle, altsetting))
```

Comment:
It's still there... Sadly the diff in github isn't great. See line 489/495 (old/new).